advanced_tutorials/electricity/1_calendar_fg.ipynb

```diff
@@ -0,0 +1,651 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c6e9cf21",
+   "metadata": {},
+   "source": [
+    "## <span style=\"color:#ff5f27;\"> 📝 Imports</span>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "3678889f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import random\n",
+    "from datetime import datetime\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "4fe3c843",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'hdfs://rpc.namenode.service.consul:8020/Projects/electricity/'"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
We shouldn't include hdfs in any of the tutorials.
If we include hdfs, the notebooks will only run inside Hopsworks.
We want notebooks to run outside hopsworks.